draft-ietf-tls-tls13.md

```diff
@@ -1854,14 +1820,6 @@ subsequent connections.
 The "key_share" extension contains the endpoint's cryptographic parameters
 for non-PSK key establishment methods (currently DHE or ECDHE).
 
-Clients which offer one or more (EC)DHE cipher suites MUST send this
-extension and SHOULD send at least one supported KeyShareEntry value.
-Servers MUST NOT negotiate any of these cipher suites unless a supported
-value was provided.
-If this extension is not provided in a ServerHello or ClientHello,
```

Comment:
Right. @martinthomson given the way that your post-handshake server auth draft is heading, I'd like to go back to the idea of using SignatureAlgorithms to indicate what the server did. Objections?